java-bootcamp-bahia-t1/src/Payment.java

```diff
@@ -0,0 +1,9 @@
+/*Interface to represent a general payment*/
+
+public interface Payment {

```

Comment:
Strategy, becouse we have different payments methods wich requiere differents algorithims. Then the transaction is abstracted from the method of payment that is to be used